### ISSUE TITLE
Large concurrent text cutoff for more than one line

### ISSUE THREAD

**Issue by ajoshi0** [ShinyChang/React-Text-Truncate#53]
**example:**
```
<div style="200px">
   <TextTruncate line={3} text="LoremIpsumissimplydummytextoftheprintingand typesettingindustry.fuhefjfvfjfkvjkvhjkjkvhjjhdfvjfhvjvj hfvjvfhjjhfjsfhjf ghj" />
</div>
```

**Output**
![screen shot 2017-07-11 at 2 40 10 pm](https://user-images.githubusercontent.com/3833185/28092021-d3a9d35a-6646-11e7-806c-a8d63fd1e7a8.png)

Currently i am using version `0.8.3` but i tested all version till `0.11.0` still issue is there in all
Any idea how should we fix this @ShinyChang

**Comment by ShinyChang** [comment]
Hi @ajoshi0 , I published `0.11.2`, which can resolve truncate incorrect in small size of width. But can not resolve this issue yet. Your test case's word too long, I can not handle it well for now. You can see my [demo page](https://shinychang.net/React-Text-Truncate/) result **6. Long words inside a small container**. Hope this can help you for now. I will keeping update this component if I have enough time.

**Comment by ShinyChang** [comment]
Fixed, please upgrade to 0.13.0
